earth_enterprise/src/common/test_plugins/FileAccessorExecutableTest.cpp

```diff
@@ -0,0 +1,21 @@
+// Copyright 2020 The Open GEE Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
Can you write a comment here that explains how this works? It appears to be a rather useless source file, but I'm sure there is some goodness hiding in here somewhere :)

Reply:
It's incredibly useful if someone wants an executable that assigns two variables to each other ;)

Point taken, I'll add comments to this and the other files in the `test_plugins` directory. This one in particular is just to produce an executable that is safe for the plugin loader to attempt, and fail, to load with `dlopen`.